Misc/NEWS.d/next/Tests/2019-01-10-09-14-58.bpo-35704.FLglYo.rst

```diff
@@ -0,0 +1,3 @@
+Prevent ''test_shutil.test_unpack_archive_xztar'' from MemoryError on 32-bit AIX
```

Comment:
why double single quote? and not a double back quote?

Reply:
Heh, I think the edit and your comment crossed in flight :)

(I thought I'd already reviewed the NEWS entry on the original PR, but apparently not)